README.md

```diff
@@ -292,20 +294,37 @@ Only the release should merge into the main branch!!!
 [![@abalanovsky](https://avatars.githubusercontent.com/u/108689551?v=4)](https://github.com/abalanovsky)
 [![@OlyaKorchan](https://avatars.githubusercontent.com/u/17857767?v=4)](https://github.com/OlyaKorchan)
 [![@MariaAfonina](https://avatars.githubusercontent.com/u/104988390?v=4)](https://github.com/MariaAfonina)
-[![@]()]()
+[![@Marichka0406](https://avatars.githubusercontent.com/u/121502737?v=4)](https://github.com/Marichka0406) 
```

Comment:
Something wrong with this picture

<img width="951" alt="Screenshot 2023-12-15 at 12 05 13" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/104988390/a9fb7a24-0616-4350-9791-a636b205c670">

Reply:
this bug has happened before. 
It will dissapear after merging i suppose

Reply:
Okay, but check this after merging, please.

Reply:
Deal